src/main/js/src/components/ExamSessionForm/ExamSessionForm.module.css

```diff
@@ -32,6 +32,7 @@ textarea {
   border: 1px solid hsl(0, 0%, 68%);
   border-radius: 2px;
   background-color: hsl(0, 0%, 100%);
+  font-size: 14px;
```

Comment:
Decided to have a bigger font size for the select element text given that way the element will probably stand out more. Perhaps even a bigger font size could be used though? This 14px was used for some other elements in that view so I felt it could make sense here as well.